AmplifyPlugins/API/APICategoryPlugin.xcodeproj/xcshareddata/xcschemes/GraphQLWithUserPoolIntegrationTests.xcscheme

```diff
@@ -24,7 +24,7 @@
             </BuildableReference>
             <SkippedTests>
                <Test
-                  Identifier = "GraphQLWithUserPoolIntegrationTests/testSetUpOnce()">
+                  Identifier = "GraphQLAuthDirectiveIntegrationTests">
```

Comment:
Within the UserPool integ test target, we're enabling just one of the two backends for testing, which is why GraphQLAuthDirectiveIntegrationTests is disabled here. It uses different set of backend and auth rules and will need some significant refactor such as 
- If continue to use V1, then it should be fairly straight foward to enable, then we might as well consolidate it to one backend with two models.  By consolidating it, we can remove section in the readme (in GraphQLWithUserPoolIntegrationTests/AuthDirective/README.md) that specifics a credentials file with username/password since the first backend adds presign-up hooks to confirm the user, and has runtime methods to register and sign in users.
- If migrating to V2, the operations on the auth rule is deny by default so update it to include "read" operation for the tests to pass, or update the tests.

considering these changes, will keep it disbabled while enabling one more integration test in the pipeline